arborist/engine.go

```diff
@@ -8,6 +8,8 @@ package arborist
 
 import (
 	"fmt"
+	"log"
```

Comment:
I recommend to use glog https://godoc.org/github.com/golang/glog. It is very elegant and easy to use